biomaj/workflow.py

```diff
@@ -1676,7 +1674,7 @@ def wf_uncompress(self):
                 elif origFile.endswith('.zip'):
                     is_archive = True
                 else:
-                    logging.warn('Uncompress: Unknown file extension')
+                    raise Exception("Workflow:wf_uncompress: Unknown file extension in '" + str(archivefile) + "'")
```

Comment:
We must not raise an exception here
We loop over download files (all of them), not being an archive is fine, we may download non compressed files.
In fact, we should not even log the unknown file extension (a .fasta is not an error). In this loop we just try to find which files are archives